cmd/kpod/stop.go

```diff
@@ -54,26 +62,41 @@ func stopCmd(c *cli.Context) error {
 
 	logrus.Debugf("Stopping containers with timeout %d", stopTimeout)
 
+	var filterFuncs []libpod.ContainerFilter
+	var containers []*libpod.Container
 	var lastError error
-	for _, container := range c.Args() {
-		ctr, err := runtime.LookupContainer(container)
+
+	if c.Bool("all"){
+		// only get running containers
+		filterFuncs = append(filterFuncs, func(c *libpod.Container) bool {
+			state, _ := c.State()
+			return state == libpod.ContainerStateRunning
```

Comment:
Are you allowing 'paused' containers to be stopped?  If so, then will this get them too?  If you can't stop a paused container, then I think the man page needs to be touched up to state that.

Reply:
We do not currently allow the stopping of a pause container.